source/geometries/Next100TrackingPlane.cc

```diff
@@ -61,69 +66,74 @@ void Next100TrackingPlane::Construct()
-                        copper_plate_name);
+  G4LogicalVolume* copper_plate_logic =
+    new G4LogicalVolume(copper_plate_solid,
+                        G4NistManager::Instance()->FindOrBuildMaterial("G4_Cu"), copper_plate_name);
 
-  G4double zpos = GetELzCoord() - gate_tp_dist_ - copper_plate_thickness_/2.;
+  G4double copper_plate_zpos = GetELzCoord() - gate_tp_dist_ - copper_plate_thickness_/2.;
 
-  G4VPhysicalVolume* copper_plate_phys_vol =
-    new G4PVPlacement(nullptr, G4ThreeVector(0.,0.,zpos),
-                      copper_plate_logic_vol, copper_plate_name, mpv_->GetLogicalVolume(),
+  G4VPhysicalVolume* copper_plate_phys =
+    new G4PVPlacement(nullptr, G4ThreeVector(0.,0.,copper_plate_zpos),
+                      copper_plate_logic, copper_plate_name, mpv_->GetLogicalVolume(),
                       false, 0, false);
 
-  copper_plate_gen_ = new CylinderPointSampler2020(copper_plate_phys_vol);
+  copper_plate_gen_ = new CylinderPointSampler2020(copper_plate_phys);
+
 
   // SIPM BOARDS /////////////////////////////////////////////////////
 
   sipm_board_geom_->SetMotherPhysicalVolume(mpv_);
   sipm_board_geom_->Construct();
-  G4LogicalVolume* sipm_board_logic_vol = sipm_board_geom_->GetLogicalVolume();
-
-  zpos = GetELzCoord() - gate_tp_dist_ + sipm_board_geom_->GetThickness()/2.;
+  G4LogicalVolume* sipm_board_logic = sipm_board_geom_->GetLogicalVolume();
 
-  // SiPM boards are positioned bottom (negative Y) to top (positive Y)
-  // and left (negative X) to right (positive X).
+  G4double zpos = GetELzCoord() - gate_tp_dist_ + sipm_board_geom_->GetThickness()/2.;
 
   G4int board_index = 1;
+  PlaceSiPMBoardColumns(5, -3.5, zpos, board_index, sipm_board_logic);
+  PlaceSiPMBoardColumns(7, -2.5, zpos, board_index, sipm_board_logic);
+  PlaceSiPMBoardColumns(8, -1.5, zpos, board_index, sipm_board_logic);
+  PlaceSiPMBoardColumns(8, -0.5, zpos, board_index, sipm_board_logic);
+  PlaceSiPMBoardColumns(8,  0.5, zpos, board_index, sipm_board_logic);
+  PlaceSiPMBoardColumns(8,  1.5, zpos, board_index, sipm_board_logic);
+  PlaceSiPMBoardColumns(7,  2.5, zpos, board_index, sipm_board_logic);
+  PlaceSiPMBoardColumns(5,  3.5, zpos, board_index, sipm_board_logic);
+
+
+  ///// DB PLUGS
+  G4double plug_x = 40. * mm;
+  G4double plug_y =  5. * mm;
+  G4double plug_z =  2. * mm;
+  G4double plug_y_displacement = 90.5 * mm;
+
+  G4Box* plug_solid = new G4Box("DB_PLUG", plug_x/2., plug_y/2., plug_z/2.);
+  G4LogicalVolume* plug_logic = new G4LogicalVolume(plug_solid,  materials::PEEK(), "DB_PLUG");
+  G4double plug_posz = copper_plate_zpos - copper_plate_thickness_/2. - plug_z/2.;
+
+  G4ThreeVector pos;
+  for (int i=0; i<int(board_pos_.size()); i++) {
+    pos = board_pos_[i];
+    pos.setY(pos.getY()-plug_y_displacement);
+    pos.setZ(plug_posz);
+    plug_pos_.push_back(pos);
+    new G4PVPlacement(0, pos, plug_logic, "DB_PLUG", mpv_->GetLogicalVolume(), false, i, false);
+  }
 
-  // Column on the far left has 5 boards.
```

Comment:
Is there any reason why you removed the comments? I think they are useful.

Reply:
Probably I though that the function calls were clear enough like `PlaceSiPMBoardColumns(5, -3.5, zpos, board_index, sipm_board_logic)` is already saying that you place 5 boards and the position. However if you think is clearer with the comments I can recover them.

Reply:
You need to go to the function and understand what it does. I think the comments help.

Reply:
Previous comments added